index.html

```diff
@@ -2484,8 +2483,8 @@ <h3>
           <li>If <a>Type</a>(<var>value</var>) is not String, return
           <code>undefined</code>.
           </li>
-          <li>Otherwise, parse <var>value</var> as if it was a [[HTML]]
-          {{HTMLLinkElement/sizes}} attribute and let <a>list</a>
+          <li>Otherwise, parse <var>value</var> as if it was a
+          {{HTMLLinkElement/sizes}} attribute and let [=list=]
```

Comment:
I don't know how you got this to work... I literally made this change yesterday on my local copy and it couldn't find Infra. Or maybe I didn't, and there's a difference between `<a>` and `[=`.

Reply:
Oh, I see your note that you thought `[[HTML]]` was confusing it.

Reply:
We take the [[CITATION]] into consideration when doing the matching (we dynamically move the citation into the nearest parent element to give "context"), which may be confusing it. Something weird there tho, so we should double check what's happened... specially for [=list=].